app/views/idv/agreement/show.html.erb

```diff
@@ -17,7 +17,8 @@
 <p><%= t('doc_auth.info.secure_account') %></p>
 
 <%= simple_form_for(
-      :doc_auth,
+      @consent_form,
+      as: :doc_auth,
```

Comment:
Curious if we could use this opportunity to drop the name alias. Where do we still rely on it?

Is it just this that needs to be updated? :

https://github.com/18F/identity-idp/blob/9a7b7d5c76f50115d93e6e6ec095d487abfd10c7/app/controllers/idv/agreement_controller.rb#L58-L60

Reply:
Yeah I'd like to drop the alias too. I've got another field name change here in flight already (starting with #9286)--it's complicated by the 50/50 state during deployment. I will probably try to remove the alias in a separate set of PRs

Reply:
Ah, right, that definitely makes things more complicated. Glad that the ball is already rolling on that at least 👍